modules/service/src/main/resources/app.conf

```diff
@@ -90,6 +90,9 @@ app {
 
   # Collection of prefixes used throughout the service
   prefixes {
+    # Nexus core context definition
+    core-context = ${app.http.public-uri}"/"${app.http.prefix}"/contexts/nexus/core/resource/v1.0.0"
```

Comment:
We probably don't need that, but just the link to `standards` context, `links` context and `search` context